### ISSUE TITLE
One friend appears more than once 

### ISSUE THREAD

**Issue by lionello** [lhchavez/gtalkchat#150]
In the contact list, one of my friends appears more than once. Same name, same contact picture, same chat history when tapped.
Interestingly, this friend is the only contact that appears immediately, while gchat+ is downloading the contacts after startup. (Which sometimes takes long, btw)
If I had to guess, I'd say this friend is the first one I ever added.

**Comment by atapetrone** [comment]
Same to me

**Comment by loofmodnar** [comment]
That's an accurate description. For me it's always the first person on my list.

**Comment by andborges** [comment]
Very, very same to me!

**Comment by andborges** [comment]
Is that project  still "alive"?

**Comment by limitedmage** [comment]
Yes, it's still alive! We've just been very busy with other things (work :( ) and haven't had the time to fix bugs lately.

**Comment by sdubrick** [comment]
Happens to me too. The weird thing is that one version they are 'away' and the other they are 'available'
